ballerina/tests/unit_tests.bal

```diff
@@ -282,8 +282,8 @@ type Member record {
 
 @test:Config{}
 public isolated function testRecordWithNil() returns error? {
-    Contact phone1 = {mobile: "+123456", home: "789"};
-    Member member1 = {name: "foo", salary: 1.23, contact: phone1};
+    // Contact phone1 = {mobile: "+123456", home: "789"};
+    // Member member1 = {name: "foo", salary: 1.23, contact: phone1};
```

Comment:
Why do we commenting these lines? (If these are no more needed, let's remove them).

Reply:
These lines are not used in assert statements hence removed now.